<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_dialog</class>
 <widget class="QDialog" name="new_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>510</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="headline_folder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1. Select folder containing images you want to label</string>
   </property>
  </widget>
  <widget class="QLabel" name="headline_num_labels">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>431</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>3. Specify labels</string>
   </property>
  </widget>
  <widget class="QLabel" name="labels_inputs_description">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>311</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>How many unique labels you want to assign?</string>
   </property>
  </widget>
  <widget class="QLabel" name="selected_folder_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>631</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_button">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_num_labels">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>270</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numLabelsInput">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>270</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="radio_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>431</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>2. Select mode</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="csv_radioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>701</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>csv (Images in selected folder are labeled and then csv file with assigned labels is &amp;generated.)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="copy_radioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>711</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>cop&amp;y (Creates folder for each label. Labeled images are copied to these folders. Csv is also generated)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="move_radioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>721</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>move (Creates folder for each &amp;label. Labeled images are moved to these folders. Csv is also generated)</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scroll_area">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>251</width>
     <height>281</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scroll_area_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>249</width>
      <height>279</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>new_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
